### PR TITLE
Added PdfContent

### PR DESCRIPTION
**Pull request by lunaroyster**
We need it:

* if we chose to host PDFs users embed
* to fetch metadata and process PDFs on our end

### CODE REVIEW

**Comment by lunaroyster** [comment]
(Whoops, didn't mean to add 880132e)

File: packages/playground/src/components/PDF.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+       :src="content.data.url"
+       width="1000"
+       height="700">
+  </div>
+</template>
+
+<script lang="ts">
+import ContentMeta from "../types/ContentMeta";
+
+// const downloader = `https://us-central1-openmindstandard.cloudfunctions.net/api/pdf/download`;
```
Comment (ArsalaBangash) [inline packages/playground/src/components/PDF.vue]: Please remove the comments from this file.

File: packages/playground/.storybook/stories/pdf.stories.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { action } from '@storybook/addon-actions'
+import { linkTo } from '@storybook/addon-links'
+import { withKnobs, text, boolean, number } from "@storybook/addon-knobs";
+
+import '../../src/plugins/axios.ts';
+
+import PDFContent from '../../src/components/PDF.vue'
```
Comment (ArsalaBangash) [inline packages/playground/.storybook/stories/pdf.stories.js]: Please rename PDF.vue to PdfContent.vue to maintain uniformity.

File: packages/playground/src/components/URLContent.vue [hunk 3]
```diff
@@ -33,25 +33,36 @@ export default {
...
-      method: "POST",
-      data: {
-        url
-      }
-    });
-    this.$data.ogData = data.data;
-    this.$data.loaded = true;
+    await this.updateUrl();
+  },
+  watch: {
```
[the remark this answers is not in the document]
Comment (lunaroyster) [reply packages/playground/src/components/URLContent.vue]: I wanted the knobs on storybook to work so we could try loading different URLs/PDFs in our components. I discovered that we needed to watch for changes to content.

If we wanted to make these viewer components editable, they will need to stay in sync with the latest state of `content`.

File: packages/playground/src/components/URLContent.vue [hunk 4]
```diff
@@ -21,7 +21,7 @@
 <script lang="ts">
 import ContentMeta from "../types/ContentMeta";
 
-const scraper = `http://us-central1-openmindstandard.cloudfunctions.net/api/opengraph/scrape`;
+const scraper = `https://us-central1-openmindstandard.cloudfunctions.net/api/opengraph/scrape`;
 
 interface URLData {
```
[the remark this answers is not in the document]
Comment (lunaroyster) [reply packages/playground/src/components/URLContent.vue]: Oh okay. Makes sense, I'll fix.

**Pull request merged**